lib/shutdown_manager.js

```diff
@@ -89,9 +89,7 @@ export default class ShutdownManager extends EventEmitter {
     this.taskListener.on('working', this.onWorking);
 
     // Kick off the idle timer if we started in an idle state.
-    if (taskListener.pending === 0) {
-      this.onIdle();
-    }
+    if (taskListener.isIdle()) this.onIdle();
```

Comment:
So I don't care... but coding style guide is unambiguous on the subject:
https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Coding_Style

Then again, I just realized I've consistently done connectives wrong... But at least I know now...